tests/test_helpers/helpers.ts

```diff
@@ -493,3 +494,16 @@ export function getCellsObject(model: Model, sheetId: UID): Record<string, CellV
   }
   return cells;
 }
+
+export function zoneToRect(
+  model: Model,
```

Comment:
isn't the getter  `getVisibleRect` enough for your test ?

Reply:
Works too. It's hard to keep track of all the getters/helpers.